test/Dialect/FIRRTL/SFCTests/constant-data-taps.fir

```diff
@@ -0,0 +1,26 @@
+; RUN: firtool --verilog %s | FileCheck %s
+
+circuit ConstantSinking : %[[
+  {
+    "class": "sifive.enterprise.grandcentral.DataTapsAnnotation",
+    "keys": [
+      {
+        "class": "sifive.enterprise.grandcentral.ReferenceDataTapKey",
+        "source": "~ConstantSinking|ConstantSinking>w",
+        "wireName": "~ConstantSinking|ConstantSinking>t"
+      }
+    ]
+  },
+  {
+    "class":"firrtl.transforms.DontTouchAnnotation",
+    "target":"~ConstantSinking|ConstantSinking>t"
+  }
+]]
+  module ConstantSinking:
+    output out : UInt<1>
+    wire t : UInt<1>
+    out <= t
+    t is invalid
+    node w = UInt<1>(1)
+
+; CHECK: wire t = 1'h1;
```

Comment:
The fact that this "just works" is a good indication that we've arrived at a reasonable design point with ref types. 💯 

I can't overemphasize how cool this is compared with the heroics that had to happen on the SFC side to get this thing to work.